### ISSUE TITLE
setting file to custom name in tab_model() not working

### ISSUE THREAD

**Issue by gtatters** [strengejacke/sjPlot#396]
when using sjt.lm or sjt.lmer, the file="test.html" works fine and creates an output file.  But these functions are deprecated and so when using tab_model() with a lmerMod fit, I get the following error when I set file="test.html":

> Error in object[[name, exact = TRUE]] : subscript out of bounds

If I leave file unassigned, the html viewer works fine and the table output is well formatted. 

Operating within Rstudio on Mac OSX, R version 3.5.1.

is there a trick to specifying a custom file name?

**Comment by gtatters** [comment]
here is a reproducible example:  
```
data(efc)
 m1<-lm(barthtot ~ c12hour, data=na.omit(efc))
 tab_model(m1, file="test.html")
```
which gives:

> Error in object[[name, exact = TRUE]] : subscript out of bounds

**Comment by strengejacke** [comment]
Yes, the file-argument is not implemented in the current version on CRAN, but it's available in the GitHub-version. Use `devtools::install_github("strengejacke/sjPlot")` to install.

**Comment by gtatters** [comment]
looks like I have all the github dependencies installed, but have that new issue above preventing the table from being generated.

**Comment by strengejacke** [comment]
I think you may need to install _sjstats_ from GitHub.

**Comment by strengejacke** [comment]
Strange, must be a server-issue, I think. I'm compiling the source-version of this package and will attach it here once it's done. In a few minutes or so.

**Comment by gtatters** [comment]
ok.  i'll let you know how I make out with the install and the original question re: file='test.html'

**Comment by strengejacke** [comment]
[sjPlot_2.5.0.9000.tar.gz](https://github.com/strengejacke/sjPlot/files/2279271/sjPlot_2.5.0.9000.tar.gz)

**Comment by gtatters** [comment]
I managed to get all libraries installed eventually, so it sounds like a server issue which is resolved.  

I can now confirm that the file option is working, using the following:  

```
library(sjPlot)
x<-rnorm(10)
y<-rnorm(10)
lm1<-lm(y~x)
setwd("~/Desktop")
tab_model(lm1, title="Summary of y ~ x", file='test.html')

```

Interesting, I found a little glitch in the process.  If you set title="Summary of y ~ x", the table is generated with a proper title:

![summary2](https://user-images.githubusercontent.com/11757426/43978497-6b10e81e-9cb5-11e8-881e-51049edeefed.jpeg)

But if you set title='summary of y ~ x' (lower case s on summary) you get:

![summary](https://user-images.githubusercontent.com/11757426/43978524-8ab6f50a-9cb5-11e8-8053-bd0c57252c7f.jpeg)

This problem only occurs with file output, and does not appear in the IDE webviewer in Rstudio.

Thank you for your help!  I'll just have to use uppercase Summary.
